src/parsers/urdf/model.hxx

```diff
@@ -365,7 +365,10 @@ inline void parseTree (::urdf::LinkConstPtr link, Model & model, const SE3 & pla
         {
           model.appendBodyToJoint(parent_joint_id, jointPlacement, Y, link->name); //Modify the parent inertia in the model
         }
-        
+        else
+        {
+          model.addFrame(link->name, parent_joint_id, nextPlacementOffset, BODY);
```

Comment:
line 376 we add the frame corresponding to the fixed_joint.
This line is for the link that follows the fixed_joint